components/monitoring/grafanadatasource_prometheus.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDataSource
+metadata:
+  name: prometheus-local-grafanadatasource
```

Comment:
I think putting they keyword customer in here might help, so we can easily tell which Prometheus data source it is.

Reply:
updated